core/src/main/java/com/sequenceiq/cloudbreak/converter/BlueprintRequestToBlueprintConverter.java

```diff
@@ -77,9 +77,10 @@ private void validateBlueprintStackVersion(Blueprint blueprint) {
         try {
             JsonNode root = JsonUtil.readTree(blueprint.getBlueprintText());
             String stackVersion = blueprintUtils.getBlueprintStackVersion(root);
-            if (StringUtils.isBlank(stackVersion) || !stackVersion.matches("[0-9]+\\.[0-9]+")) {
-                throw new BadRequestException(String.format("Stack version [%s] is not valid. Valid stack version is in MAJOR.MINOR format eg.: 2.6",
-                        stackVersion));
+            if (StringUtils.isBlank(stackVersion) || !stackVersion.matches("[0-9]+\\.[0-9]?(\\.?[0-9])")) {
```

Comment:
Does this mean that we do not allow e.g. 2.6 anymore? Isn't it needed?

Reply:
We allow both version type. This modification is allows the three-part versions too.

Reply:
Perfect, thanks